Gemfile

```diff
@@ -63,6 +63,10 @@ end
 
 gem "rubyzip", '~> 1.2'
 
+if RUBY_VERSION.to_f < 2.5
+  gem 'sprockets', '~> 3.7.0'
```

Comment:
Won't Bundler figure it out on its own? They do specify a version:
```
  s.required_ruby_version = '>= 2.5.0'
```
shouldn't `bundle` take this into account and check out `sprockets` < 4.0 when it understands that 4+ won't work?

Reply:
This what I thought but the CI failed earlier for this PR https://github.com/rspec/rspec-rails/pull/2346. But I am not able to find it again. It was on Ruby 2.4.

I will close it for now. We will see if it happen again.